docs/blueprints/AMD_Landing_Zone.md

```diff
@@ -0,0 +1,89 @@
+AMD Landing Zone
```

Comment:
This file is not referenced anywhere at this moment. Maybe it should be added somewhere in the website's nav?

Reply:
LZ was renamed to SKL (Secure Kernel Loader).